### ISSUE TITLE
Python 3.7 compability

### ISSUE THREAD

**Issue by Tillsten** [mabuchilab/NiceLib#7]
Hey,
very nice helper library, but I think it is broken for Python 3.7 due to PEP 479. At least
I get a `RuntimeError: generator raised StopIteration` while parsing a header.

```
  File "D:\Anaconda3\lib\site-packages\nicelib\process.py", line 2080, in cdecl_hook
    for token in tokens:

RuntimeError: generator raised StopIteration
```

**Comment by natezb** [comment]
Thanks for the report. This should be fixed in 5dc3f7594a7fa51312e8183e75e70491c68348de. This appears to be a commonly-experienced bug though, so we should probably make a release soon.

Let us know if you still have this issue in the current git version.

**Comment by natezb** [comment]
FYI I just released 0.6, which should be available on PyPI.
